sdks/python/test-suites/direct/xlang/build.gradle

```diff
@@ -67,3 +69,20 @@ createCrossLanguageValidatesRunnerTask(
     "--endpoint localhost:${jobPort}",
   ],
 )
+
+createTransformServiceTask(
```

Comment:
Can we add a comment here that these tests are run with Py38?

Reply:
Actually, these test can run with all Python versions. Updated.

(JavaUsingPython will be restricted to Python 3.8 I'll mention that later when we add such tests).